streamlit_web.py

```diff
@@ -636,22 +642,45 @@ def interactive():
     if ner_cb:
         doc = az.get_nlp(input_text)
         named_entities = az.named_entity_recognization(input_text)
-        if len(named_entities) > 0:
-            html = spacy.displacy.render(doc, style="ent")
-            # Newlines seem to mess with the rendering
-            html = html.replace("\n", " ")
-            HTML_WRAPPER = """<div style="overflow-x: auto; border: 1px solid \
-        #e6e9ef; border-radius: 0.25rem; padding: 1rem; margin-bottom: 2.5rem">\
-        {}</div>"""
-            st.write(HTML_WRAPPER.format(html), unsafe_allow_html=True)
-        else:
-            st.info("No named entity recognized")
+        az.displacy_renderer(named_entities)
     if sentiment_cb:
         sentiments = TextBlob(az.lemmatized_text(input_text))
         st.write(sentiments.sentiment)
     if summary_cb:
         summaries = sz.summarize_text(input_text)
         st.write(summaries)
 
+
+def entities():
+    """Page to display entity analysis"""
+    st.write("Entity analysis inspects the given text for known entities \
+    and returns information about those entities. It is a way to extract \
+    information that seeks to locate and classify named entities in text \
+    into pre-defined categories such as the names of persons, organizations, \
+    locations, expressions of times, quantities, monetary values, and percentages.")
+
+    # make a copy of the main dataframe
+    input_df = preprocessed_df.copy(deep=True)
+
+    # list out possible user documents to choose from and select one
+    students = st.selectbox(
+        label="Select specific students below:",
+        options=input_df[stu_id].unique(),
+    )
+
+    # create a dataframe with the selected user and convert it to a string
+    df_selected_stu = input_df.loc[
```

Comment:
Perhaps also take a look at the suggestion @noorbuchi provided above. That seems like a good way to handle displaying multiple documents. Right now it's just all stacked together.